template/koa-rest-api/Dockerfile

```
@@ -31,4 +31,4 @@ ARG PORT=8001
 ENV PORT ${PORT}
 EXPOSE ${PORT}
 
-CMD ["lib/listen.js"]
+CMD ["--require", "./lib/tracing.js", "lib/listen.js"]
```

```suggestion
CMD ["--require", "lib/tracing.js", "lib/listen.js"]
```

Or the other way; I'm not fussed as long as we're consistent:

```suggestion
CMD ["--require", "./lib/tracing.js", "./lib/listen.js"]
```